Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/PropertyEditorSystem.cpp

```diff
@@ -17,6 +17,7 @@ namespace AZ::DocumentPropertyEditor
         Nodes::Reflect(this);
 
         AZ::Interface<PropertyEditorSystemInterface>::Register(this);
+        AZ::AllocatorInstance<AZ::Dom::ValueAllocator>::Create();
```

Comment:
I think this will need a corresponding `AZ::AllocatorInstance<AZ::Dom::ValueAllocator>::Destroy();` in the `PropertyEditorSystem` destructor or you might get unit test failures if you use this class in a fixture on teardown.

Reply:
good catch, thanks!